hello.go

```diff
@@ -86,13 +86,17 @@ func main() {
 
 	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
 		tmpl.Execute(w, data)
+		log.Printf("%s request from %s for %s", r.Method, r.Host, r.URL)
```

Comment:
Unclear why this was added. Cloud Run already has request logs. I suggest to remove

Reply:
Removed.